### DISCUSSION TITLE
Support for modulo operator

### DISCUSSION THREAD

**Discussion by johndagostino** [Hexagon/croner#231] [Ideas]
fugit in ruby land supports an extension to standard cron called the modulo operator

https://github.com/floraison/fugit#the-modulo-extension

Would this be something that could potentially be supported by corner?

**Comment by Hexagon** [comment]
Thanks for the suggestion! Modulo operator could be an interesting addition to croner.

Do you have any current use cases for this, apart from those mentioned in the fugit docs?

**Comment by johndagostino** [comment]
@Hexagon, sorry for the delay; as the fugit docs describe, we have a use case where we want fortnight jobs locked to a specific day. 

for example `every second Thursday at 3pm AEDT`
